test/clj/rems/test_browser.clj

```diff
@@ -315,83 +424,170 @@
+        (scroll-and-click  :create-license)
+        (et/wait-visible  {:tag :h1 :fn/text "Create license"})
+        (select-option  "Organization" "nbn")
+        (scroll-and-click :licensetype-link)
+        (fill-form-field "License name" (str (:license-name @test-context) " EN") {:index 1})
+        (fill-form-field "License link" "https://www.csc.fi/home" {:index 1})
+        (fill-form-field "License name" (str (:license-name @test-context) " FI") {:index 2})
+        (fill-form-field "License link" "https://www.csc.fi/etusivu" {:index 2})
+        (fill-form-field "License name" (str (:license-name @test-context) " SV") {:index 3})
+        (fill-form-field "License link" "https://www.csc.fi/home" {:index 3})
+        (et/screenshot (io/file reporting-dir "about-to-create-license.png"))
+        (scroll-and-click :save)
+        (et/wait-visible {:tag :h1 :fn/text "License"})
+        (wait-page-loaded)
+        (et/screenshot (io/file reporting-dir "created-license.png")))
+      (is (str/includes? (et/get-element-text driver {:css ".alert-success"}) "Success"))
+      (is (= {"Organization" "nbn"
+              "Title (EN)" (str (:license-name @test-context) " EN")
+              "Title (FI)" (str (:license-name @test-context) " FI")
+              "Title (SV)" (str (:license-name @test-context) " SV")
+              "Type" "link"
+              "External link (EN)" "https://www.csc.fi/home"
+              "External link (FI)" "https://www.csc.fi/etusivu"
+              "External link (SV)" "https://www.csc.fi/home"
+              "Active" ""}
+             (slurp-fields driver :license))))))
+
+(defn test-create-resource []
+  (let [driver (get-driver)]
+    (et/with-postmortem driver {:dir reporting-dir}
+      (doto driver
+        (go-to-admin-resources)
+        (scroll-and-click :create-resource)
+        (et/wait-visible {:tag :h1 :fn/text "Create resource"})
+        (select-option "Organization" "nbn")
+        (fill-form-field "Resource identifier" (:resid @test-context))
+        (select-option "License" (str (:license-name @test-context) " EN"))
+        (et/screenshot (io/file reporting-dir "about-to-create-resource.png"))
+        (scroll-and-click :save)
+        (et/wait-visible {:tag :h1 :fn/text "Resource"})
+        (wait-page-loaded)
+        (et/screenshot (io/file reporting-dir "created-resource.png")))
+      (is (str/includes? (et/get-element-text driver {:css ".alert-success"}) "Success"))
+      (is (= {"Organization" "nbn"
+              "Resource" (:resid @test-context)
+              "Active" ""}
+             (slurp-fields driver :resource)))
+      (is (= (str "License \"" (:license-name @test-context) " EN\"")
+             (et/get-element-text driver [:licenses {:class :license-title}]))))))
+
+(deftest test-create-catalogue-item
+  (let [driver (get-driver)]
+    (et/with-postmortem driver {:dir reporting-dir}
+      (login-as driver "owner")
+      (swap! test-context assoc
```

Comment:
since you're actually calling `test-create-license`, why not just pass the license name in?

Reply:
I was trying to simplify calling these functions manually one by one. Let's test and see if this is workable or not.